sos/report/plugins/mvcli.py

```diff
@@ -0,0 +1,31 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+
+# This sosreport plugin is meant for sas adapters.
+# This plugin logs inforamtion on each adapter it finds.
+
+from sos.report.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class mvCLI(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+
+    short_desc = 'mvCLI Integrated RAID adapter information'
+
+    plugin_name = "mvcli"
+    commands = ("/opt/marvell/bin/mvcli",)
+
+    def setup(self):
+
+        # get list of adapters
+        self.collect_cmd_output("/opt/marvell/bin/mvcli info -o vd", timeout=5)
+        self.collect_cmd_output("/opt/marvell/bin/mvcli info -o pd", timeout=5)
+        self.collect_cmd_output("/opt/marvell/bin/mvcli info -o hba", timeout=5)
+        self.collect_cmd_output("/opt/marvell/bin/mvcli smart -p 0", timeout=5)
```

Comment:
Same here: use a single call and don't use `collect_cmd_output()` unless you are going to do something with the result.

Reply:
ok  sure..will make changes